### ISSUE TITLE
Rules should be plain callables

### ISSUE THREAD

**Issue by tuupola** [tuupola/slim-jwt-auth#163]
Can be backported from https://github.com/tuupola/branca-middleware/pull/16.

**Comment by MrPropre** [comment]
@tuupola Do you plan to update your library to implement this change ? Thank you in advance.

**Comment by tuupola** [comment]
Yes it is still the plan. Probably for 4.x though, since it can be considered a bc break.
